content/en/community/end-user/discussion-group.md

```diff
@@ -5,46 +5,8 @@ description: Monthly OpenTelemetry usage discussions, all are welcome!
 weight: 20
 ---
 
-**As of April 2024, AMER and EMEA sessions are discontinued in favor of
-discussion panels with a more targeted focus.**
+**As of April 2024, discussion group sessions are discontinued in favor of
+end-user panels with a more targeted focus.**
 
-Interested in learning how other end users are implementing OpenTelemetry in
-their organizations? Come talk shop with other OpenTelemetry end users! This is
-a vendor-neutral space to discuss best practices, ask questions, and meet other
-observability-minded folks.
-
-Feedback that is shared and collected in these sessions will be routed back to
-the relevant project maintainers to help drive prioritization of improvements
-and changes to the project.
-
-## About
-
-- Search the blog for "End-User Discussions" to find summaries of past
-  discussions.
-- A project maintainer and/or a Governance Committee member will be in
-  attendance at each session to provide additional context, insight, and plans
-  to user questions and feedback.
-- Sessions are recorded.
-
-## Upcoming sessions
-
-Here are upcoming sessions, or you can view them on the
-[OpenTelemetry calendar](https://github.com/open-telemetry/community#calendar):
-
-- **APAC (Asia Pacific)**: every third Wednesday of the month at 11AM India ST
-  (GMT +5.5), [register](https://lu.ma/1w129wgu) to get the Zoom link
-  - April 17
-  - May 15
-  - June 19
-
-Past topics/questions have included:
-
-- Best practices on monitoring collectors in production
-- Using OTel in CI/CD pipelines
-- What’s holding you back from adopting more OpenTelemetry?
-- Auto vs manual instrumentation use cases
-- How to secure a publicly exposed OTel Collector?
-
-We use a [Lean Coffee](https://leancoffee.org) format where discussion topics
-are generated and democratically selected by the group at the start of the
-meeting. Topics are rigorously time-boxed by a facilitator.
+Check out https://github.com/open-telemetry/sig-end-user if you would like
+to participate or help organize a future end-user panel.
```

Comment:
I'm not sure how helpful this last sentence is by redirecting the reader to a repo. What is an interested person meant to do? Create an issue?

Maybe it would be better to direct folks to [#otel-sig-end-user](https://cloud-native.slack.com/archives/C01RT3MSWGZ)?

Reply:
> What is an interested person meant to do? Create an issue? 

That's the idea, but it requires more docs on the repo itself. We'd like maintainers to be able to propose an end-user panel/feedback session by creating an issue on that repo. However, considering this process has not been documented yet, I think you're right and pointing them to the Slack channel is less confusing.

Reply:
@open-telemetry/sig-end-user-maintainers what do you think? My concern with Slack is that not all end-users are on Slack, and some are not allowed by their organisation contribution policies to join. So, if we point to the end-user repo we can later explain there how to get involved in user feedback sessions?

Reply:
I've updated this to include both, link to repo and Slack